scripts/setup-network.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+iptables -I INPUT -m tcp -p tcp -m multiport --dports 443,8443 -d 192.168.42.0/24 -j ACCEPT
```

Comment:
Is minishift guaranteed to take this IP?

Reply:
I've never seen it on another subnet, but I can parameterize it to allow someone to override it easily.